rpgsaga/saga/src/cat.ts

```diff
@@ -0,0 +1,46 @@
+export class Cat {
+  private theAge: number;
+  private theName: string;
+  private theBreed: string;
+  private theWeight: number;
+
+  constructor(age: number, name: string, breed: string, weight: number) {
+    this.theAge = age;
+    this.theName = name;
```

Comment:
```suggestion
    this.Name = name;
```
чтобы вызывался соответствующий set для данной переменной